datafusion/core/tests/sql/aggregates.rs

```diff
@@ -401,3 +401,83 @@ async fn test_first_value_with_sort() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn last_value() -> Result<()> {
+    let session_ctx = SessionContext::new();
+    session_ctx
+        .sql("CREATE TABLE abc AS VALUES (1, 2, 3), (null, 5, 6)")
```

Comment:
I think this kind of test are easier to maintain in `datafusion/sqllogictest/test_files/aggregate.slt`

Reply:
Removed. I have also removed `first_value` test I added last time